.github/workflows/release-charts.yaml

```diff
@@ -0,0 +1,118 @@
+# Workflow to tag and publish swan/ and swan-cern/ charts in this repository
```

Comment:
This is the main workflow that orchestrates everything, right? I would say it explicitly here. And I would list all the steps it orchestrates, here you only mention the tagging and commiting of new versions to the github repos, but there's also check_branch, lint, list_changed_charts and tag charts (which also including pushing to harbor with the corresponding secrets) with the dependency update in between.